swo_aws_extension/aws/client.py

```
@@ -468,21 +514,33 @@ def _extract_linked_account_keys(cost_and_usage: list[dict]) -> list[str]:
 
 
-def get_linked_accounts_with_usage(
+def _build_account_names_map(dimension_attributes: list[dict]) -> dict[str, str]:
+    """Build a mapping of account ID to account name from DimensionValueAttributes."""
+    return {
+        attr.get("Value", ""): attr.get("Attributes", {}).get("description", "")
+        for attr in dimension_attributes
+    }
+
+
+def get_linked_accounts_with_usage(  # noqa: WPS210
```

Ruff (0.15.17)

[warning] Invalid rule code in `# noqa`: WPS210

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)